dotnet/src/webdriver/PrintOptions.cs

```diff
@@ -50,12 +50,24 @@ public class PrintOptions
         private const double DefaultPageHeight = 21.59;
         private const double DefaultPageWidth = 27.94;
         private const double CentimetersPerInch = 2.54;
-
+        public static PageSize A4 { get; } = new PageSize { Width = 21.0, Height = 29.7 }; // cm
+        public static PageSize Legal { get; } = new PageSize { Width = 21.59, Height = 35.56 }; // cm
+        public static PageSize Letter { get; } = new PageSize { Width = 21.59, Height = 27.94 }; // cm
+        public static PageSize Tabloid { get; } = new PageSize { Width = 27.94, Height = 43.18 }; // cm
         private double scale = 1.0;
         private PageSize pageSize = new PageSize();
         private Margins margins = new Margins();
         private readonly HashSet<object> pageRanges = new HashSet<object>();
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PrintOptions"/> class with default values.
+        /// Default page size is set to A4.
+        /// </summary>
+        public PrintOptions()
+        {
+        this.PageDimensions = A4; // Default to A4 page size
```

Comment:
The default is already set, don't overwrite it here. Moreover, this value is optional, which means that default value is determined on remote end. As a client, we should send values only if user specifies.